fusor-installer.spec

```diff
@@ -44,13 +45,17 @@ the Fusor Foreman plugin
 # convert manpages
 a2x -d manpage -f manpage fusor-installer.8.asciidoc
 a2x -d manpage -f manpage fusor-setup.8.asciidoc
+a2x -d manpage -f manpage fusor-update-repos.8.asciidoc
+a2x -d manpage -f manpage fusor-upgrade-qci.8.asciidoc
 
 %install
 install -d -m0755 %{buildroot}%{_datadir}/fusor-installer
 cp -R hooks modules %{buildroot}%{_datadir}/fusor-installer
 install -d -m0755 %{buildroot}%{_sbindir}
 cp bin/fusor-installer %{buildroot}%{_sbindir}/fusor-installer
 cp bin/fusor-setup %{buildroot}%{_sbindir}/fusor-setup
+cp bin/fusor-update-repos %{buildroot}%{_sbindir}/fusor-update-repos
```

Comment:
Since we don't expect end users to ever call fusor-update-repos I'd suggest we put it in another location that typically isn't on the path, perhaps  /usr/libexec/fusor-installer/fusor-update-repos